### ISSUE TITLE
BatchNorm for complex tensor

### ISSUE THREAD

**Issue by nissy-shota** [pytorch/pytorch#81749]
### 🚀 The feature, motivation and pitch

I am working on building a network that takes into consider phase information in speech processing, and would like batchnorm for complex tensor to be possible.

Similar to #47052, #46642

**Comment by ezyang** [comment]
Maybe we can stand this up on top of inductor

**Comment by nissy-shota** [comment]
Thank you for your reply.
Sorry I'm not familiar with the operation.
When you say "stand this on top of the inductor", 
do you mean the "added module:primTorch label" operation you performed?

**Comment by ngimel** [comment]
cc @kshitij12345

**Comment by albanD** [comment]
Also it is not clear how statistics computation should be done in the complex space here?

**Comment by nissy-shota** [comment]
There were two types of batch normalization in complex space that I discovered in my survey. 
Are there any logical errors in the statistical calculations in complex space discussed in this paper?

3.5 [link](https://arxiv.org/abs/1705.09792)
II.C [link](https://ieeexplore.ieee.org/document/8659610)
